### ISSUE TITLE
Not compatible with Bookshelf new version

### ISSUE THREAD

**Issue by lacivert** [seegno/bookshelf-json-columns#55]
I can use this plugin with 0.x Bookshelf but when I grab 1.x, it gives me the error:

```
SyntaxError: Unexpected token o in JSON at position 1
    at JSON.parse (<anonymous>)
    at constructor.jsonColumns.forEach.column (/path/to/project/node_modules/bookshelf-json-columns/dist/index.js:49:38)
    at Array.forEach (<anonymous>)
    at Child.parse (/path/to/project/node_modules/bookshelf-json-columns/dist/index.js:47:32)
    at Promise.mapSeries (/path/to/project/node_modules/bookshelf/lib/base/events.js:101:64)
    at tryCatcher (/path/to/project/node_modules/bluebird/js/release/util.js:16:23)
    at Object.gotValue (/path/to/project/node_modules/bluebird/js/release/reduce.js:166:18)
    at Object.gotAccum (/path/to/project/node_modules/bluebird/js/release/reduce.js:155:25)
    at Object.tryCatcher (/path/to/project/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/path/to/project/node_modules/bluebird/js/release/promise.js:547:31)
    at Promise._settlePromise (/path/to/project/node_modules/bluebird/js/release/promise.js:604:18)
    at Promise._settlePromiseCtx (/path/to/project/node_modules/bluebird/js/release/promise.js:641:10)
    at _drainQueueStep (/path/to/project/node_modules/bluebird/js/release/async.js:97:12)
    at _drainQueue (/path/to/project/node_modules/bluebird/js/release/async.js:86:9)
    at Async._drainQueues (/path/to/project/node_modules/bluebird/js/release/async.js:102:5)
    at Immediate.Async.drainQueues [as _onImmediate] (/path/to/project/node_modules/bluebird/js/release/async.js:15:14)
```

**Comment by lacivert** [comment]
I have commented the foreach code block; it seems working for insert on jsonb columns. I haven't tested for other methods.

**Comment by ricardogama** [comment]
Released as [3.0.0](https://github.com/seegno/bookshelf-json-columns/releases/tag/3.0.0)
